netunnel/server/server.py

```diff
@@ -147,6 +148,24 @@ async def _setup_peers(self):
             peer = await self._load_peer_from_config(peer_config, verify_connectivity=False)
             self._peers[peer.id] = peer
 
+    async def _shutdown_peers(self):
+        """
+        Disconnect static tunnels and stop peers.
+        """
+        while self._peers:
+            _, peer = self._peers.popitem()
```

Comment:
Ummm I wonder maybe it's better to pop the peer only after deleting its tunnels.